rules/xcodeproj.bzl

```diff
@@ -1255,3 +1257,17 @@ Additional LLDB settings to be added in each target's .lldbinit configuration fi
     },
     executable = True,
 )
+
+def xcodeproj(name, use_xchammer = False, **kwargs):
```

Comment:
Did you consider to organize this in a way that we can flip it on / off with a configurable variable like other features. Then, we can test our entire CI / transparently migrate people while providing an opt-out:

```
--features rules_ios.disable_xchammer
```

To do so ATM you can probably define both rules:

```
macro
   xchammer_xcodeproj(name=name_xch, project_name=..)
   legacy_xcodeproj(name=name_legacy project_name=..)
   alias(name=name, actual=select(xch: name_xch, default=name_legacy))
````

Reply:
I didn't consider using an `alias` here! Thx this is much better, I'll use this approach instead

Reply:
Done!

Reply:
Some problems here though, the tests that run on CI often call `//...` or `//tests/ios/...` and that includes the xchammer targets now that we're adding both rules to the BUILD files, will make some changes to ignore those for now on the CI tests

Reply:
Reverted this, more details here: https://github.com/bazel-ios/rules_ios/pull/443#issuecomment-1105441681